todo_files/databaseApi.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+from bson import ObjectId
+from pymongo import MongoClient
+from datetime import datetime
+import todo_files.card as Card
+
+
+class DatabaseThing:
+
+    def __init__(self, url, db):
+        self._db = MongoClient(url)[db]
+
+    def getCardData(self):
+        allCards = Card.convertListToCards(self._db['FullList'].find())
+        return allCards
+
+    def get_card_by_id(self, id):
+        card = next((card for card in self.getCardData() if str(card.id) == id), None)
+        return card
+
+    def createCard(self, name, desc, due):
+        card = {
+            'id': ObjectId(),
```

Comment:
Mongo automatically assigns a random id to the `_id` field, so you could use that instead of making your own.